VideoApi/VideoApi/Services/ConsultationService.cs

```diff
@@ -97,7 +97,8 @@ public async Task ParticipantTransferToRoomAsync(Guid conferenceId, Guid partici
                     new GetConferenceByIdQuery(conferenceId));
             var participant = conference.GetParticipants().Single(x => x.Id == participantId);
 
-            await TransferParticipantAsync(conferenceId, participantId.ToString(), participant.GetCurrentRoom(), room);
+            var patId = participant.GetInterpreterRoom()?.Id.ToString() ?? participantId.ToString(); 
```

Comment:
pat?

Reply:
renamed to be representative of what the Id is for